disable trying to create system-wide configuration file in /etc
author: Kenneth Hoste(HPC-UGent)
--- build.orig	2015-12-12 21:51:18.238686800 +0100
+++ build	2015-12-12 21:53:23.749943093 +0100
@@ -958,10 +958,11 @@
     report ('ok' + os.linesep)
 
   if not os.path.exists (configfile):
-    report ('creating default system-wide configuration file (setting up multi-threading with ' + str(num_processors) + ' threads)... ')
-    with fopen(configfile, 'wb') as fd:
-      fd.write ('NumberOfThreads: ' + str(num_processors) + '\n');
-    report ('ok' + os.linesep)
+    #report ('creating default system-wide configuration file (setting up multi-threading with ' + str(num_processors) + ' threads)... ')
+    #with fopen(configfile, 'wb') as fd:
+    #  fd.write ('NumberOfThreads: ' + str(num_processors) + '\n');
+    #report ('ok' + os.linesep)
+    report ('WARNING not creating system-wide configuration file at %s' % configfile)
   else:
     report ('configuration file already exists - leaving as-is' + os.linesep)
 
